Model/Config/Backend/EnableProductProtection.php

```diff
@@ -43,7 +43,7 @@ public function __construct(
 
     /**
      * This will create a new Product Protection product when V2 PP is enabled
-     * If disabled, shipping protection and V2 product protection is disabled as well.
+     * If disabled, this will also disable cart offers
```

Comment:
this comment was actually already out of date - just adding a note here for what it does now (when PP is turned off, disable cart offers)